prediction_market_agent/agents/safe_guard_agent/evals/gather_latest_transaction_cases.py

```diff
@@ -0,0 +1,65 @@
+import json
+
+import pandas as pd
+import typer
+from prediction_market_agent_tooling.gtypes import ChecksumAddress
+from pydantic_evals import Case
+from tqdm import tqdm
+from web3 import Web3
+
+from prediction_market_agent.agents.safe_guard_agent.evals.models import (
+    PydanticCase,
+    SGCase,
+)
+from prediction_market_agent.agents.safe_guard_agent.safe_api_utils import (
+    get_balances_usd,
+    get_safe_detailed_transaction_info,
+    get_safe_history_multisig,
+)
+from prediction_market_agent.agents.safe_guard_agent.safe_guard import (
+    ValidationConclusion,
+)
+
+
+def main(n: int = 10) -> None:
+    """
+    Gathers testing data that should all be okay.
+    It's always latest transaction from biggest Safes that there are.
+    """
+    # Downloaded from https://dune.com/queries/4933998/8164678.
+    data = pd.read_csv(
+        "prediction_market_agent/agents/safe_guard_agent/evals/data/big_safes.csv"
+    )
+    big_safes = [Web3.to_checksum_address(addr) for addr in data["safe_address"]]
```

Comment:
why not `set([addresses])`?

Reply:
I don't understand the question 😄 `big_safes.csv` is deduplicated so doesn't really matter.